modules/nf-core/sentieon/tnhaplotyper2/tests/main.nf.test

```diff
@@ -78,7 +78,11 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out).match() }
+                { assert snapshot(
+                    path(process.out.vcf[0][1]).vcf.summary,
+                    process.out.index,
+                    process.out.versions,
```

Comment:
Could you add
`path(process.out.versions[0]).yaml`
too?

Reply:
I was getting an error from the stub test when I tried to add that there, but put it into the main tests